README.md

```diff
@@ -202,26 +239,23 @@ Marine areas are considered `common_optional`.
 
 `102371933`
 
-Things like "The Bay Area" – this one is hard so we shouldn't spend too much
-time worrying about the details yet but instead treat as something we want to do
-eventually.
+Things like "The Bay Area" – this one is hard so we shouldn't spend too much time worrying about the details yet but instead treat as something we want to do eventually.
 
-Metropolitain areas are considered `common optional`.
+Metropolitain areas are considered `common_optional`.
 
 ### microhood
 
 `102312321`
 
-Because all place is disputed. And everyone has a name for a place that will
-offend someone else.
+Because all place is disputed. And everyone has a name for a place that will offend someone else.
 
 Microhoods are considered `optional`.
 
 ### neighbourhood
 
 `102312319`
 
-Things no one will agree on. Ever.
+Neighbourhoods can represent an area surrounding a "place", a community grouping, or even a physical area. One thing is certain - neighbourhoods are things no one will agree on. Ever. 
```

Comment:
`One thing is certain - neighbourhoods are things no one will completely agree on. Ever.` (add **completely**)